source/common/protobuf/utility.h

```diff
@@ -177,6 +178,26 @@ class MessageUtil {
   static void loadFromYaml(const std::string& yaml, Protobuf::Message& message);
   static void loadFromFile(const std::string& path, Protobuf::Message& message);
 
+  template <class MessageType>
```

Comment:
I don't think this need to be templatized, using `Protobuf::Message` should be just enough.